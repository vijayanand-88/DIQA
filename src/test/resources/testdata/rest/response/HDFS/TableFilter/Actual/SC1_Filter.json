{
  "count" : 5,
  "results" : [ {
    "id" : "Default.Directory:::5515",
    "type" : "Directory",
    "name" : "ROOT",
    "asg.parentTypeNamePath" : [ "Cluster/Cluster Demo/Service/HDFS" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::12229",
      "name" : "HDFS_BA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: HDFS_BA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "originItemID" : "Default.BusinessApplication:::5447",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5448,
      "asg_modifiedby" : "Service",
      "target" : 5515,
      "originItemCatalog" : "Default"
    }, {
      "id" : "Default.tag:::12235",
      "name" : "Hadoop Files",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "Hadoop Filesystem",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 401,
      "asg_modifiedby" : "Service",
      "target" : 5515
    }, {
      "id" : "Default.tag:::12244",
      "name" : "SC1_folderwith1subfolder",
      "rootTagName" : "SC1_folderwith1subfolder",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5519,
      "asg_modifiedby" : "Service",
      "target" : 5515
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.File:::5518",
    "type" : "File",
    "name" : "testdata.csv",
    "asg.parentTypeNamePath" : [ "Cluster/Cluster Demo/Service/HDFS/Directory/ROOT/Directory/idaautomation/Directory/sub1" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::12228",
      "name" : "HDFS_BA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: HDFS_BA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "originItemID" : "Default.BusinessApplication:::5447",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5448,
      "asg_modifiedby" : "Service",
      "target" : 5518,
      "originItemCatalog" : "Default"
    }, {
      "id" : "Default.tag:::12234",
      "name" : "Hadoop Files",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "Hadoop Filesystem",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 401,
      "asg_modifiedby" : "Service",
      "target" : 5518
    }, {
      "id" : "Default.tag:::12239",
      "name" : "createDir",
      "rootTagName" : "createDir",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5520,
      "asg_modifiedby" : "Service",
      "target" : 5518
    }, {
      "id" : "Default.tag:::12243",
      "name" : "SC1_folderwith1subfolder",
      "rootTagName" : "SC1_folderwith1subfolder",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5519,
      "asg_modifiedby" : "Service",
      "target" : 5518
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.File:::5528",
    "type" : "File",
    "name" : "testdataRenamed.csv",
    "asg.parentTypeNamePath" : [ "Cluster/Cluster Demo/Service/HDFS/Directory/ROOT/Directory/idaautomation/Directory/sub1" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::12264",
      "name" : "HDFS_BA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: HDFS_BA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:15:45.418+0000",
      "asg_createdat" : "2020-10-20T15:15:45.418+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "originItemID" : "Default.BusinessApplication:::5447",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5448,
      "asg_modifiedby" : "Service",
      "target" : 5528,
      "originItemCatalog" : "Default"
    }, {
      "id" : "Default.tag:::12267",
      "name" : "Hadoop Files",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "Hadoop Filesystem",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:15:45.418+0000",
      "asg_createdat" : "2020-10-20T15:15:45.418+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 401,
      "asg_modifiedby" : "Service",
      "target" : 5528
    }, {
      "id" : "Default.tag:::12269",
      "name" : "createDir",
      "rootTagName" : "createDir",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:15:45.418+0000",
      "asg_createdat" : "2020-10-20T15:15:45.418+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5520,
      "asg_modifiedby" : "Service",
      "target" : 5528
    }, {
      "id" : "Default.tag:::12271",
      "name" : "SC1_folderwith1subfolder",
      "rootTagName" : "SC1_folderwith1subfolder",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:15:45.418+0000",
      "asg_createdat" : "2020-10-20T15:15:45.418+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5519,
      "asg_modifiedby" : "Service",
      "target" : 5528
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Directory:::5516",
    "type" : "Directory",
    "name" : "idaautomation",
    "asg.parentTypeNamePath" : [ "Cluster/Cluster Demo/Service/HDFS/Directory/ROOT" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::12230",
      "name" : "HDFS_BA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: HDFS_BA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "originItemID" : "Default.BusinessApplication:::5447",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5448,
      "asg_modifiedby" : "Service",
      "target" : 5516,
      "originItemCatalog" : "Default"
    }, {
      "id" : "Default.tag:::12236",
      "name" : "Hadoop Files",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "Hadoop Filesystem",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 401,
      "asg_modifiedby" : "Service",
      "target" : 5516
    }, {
      "id" : "Default.tag:::12240",
      "name" : "createDir",
      "rootTagName" : "createDir",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5520,
      "asg_modifiedby" : "Service",
      "target" : 5516
    }, {
      "id" : "Default.tag:::12245",
      "name" : "SC1_folderwith1subfolder",
      "rootTagName" : "SC1_folderwith1subfolder",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5519,
      "asg_modifiedby" : "Service",
      "target" : 5516
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Directory:::5517",
    "type" : "Directory",
    "name" : "sub1",
    "asg.parentTypeNamePath" : [ "Cluster/Cluster Demo/Service/HDFS/Directory/ROOT/Directory/idaautomation" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::12231",
      "name" : "HDFS_BA",
      "rootTagName" : "BusinessApplication",
      "parentTagName" : "",
      "definition" : "BusinessApplication: HDFS_BA",
      "status" : "APPROVED",
      "type" : "BusinessApplication Tag",
      "image" : {
        "icon" : "icon-business",
        "color" : "#004ea8"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "originItemID" : "Default.BusinessApplication:::5447",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5448,
      "asg_modifiedby" : "Service",
      "target" : 5517,
      "originItemCatalog" : "Default"
    }, {
      "id" : "Default.tag:::12237",
      "name" : "Hadoop Files",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "Hadoop Filesystem",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 401,
      "asg_modifiedby" : "Service",
      "target" : 5517
    }, {
      "id" : "Default.tag:::12241",
      "name" : "createDir",
      "rootTagName" : "createDir",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5520,
      "asg_modifiedby" : "Service",
      "target" : 5517
    }, {
      "id" : "Default.tag:::12246",
      "name" : "SC1_folderwith1subfolder",
      "rootTagName" : "SC1_folderwith1subfolder",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T15:14:40.525+0000",
      "asg_createdat" : "2020-10-20T15:14:40.525+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 5519,
      "asg_modifiedby" : "Service",
      "target" : 5517
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  } ],
  "configs" : {
    "defaultListView" : [ {
      "value" : "asg.parentTypeNamePath",
      "label" : "Hierarchy"
    }, {
      "value" : "asg.tagNames",
      "label" : "Tags"
    } ]
  },
  "configsMap" : {
    "*" : {
      "*" : "defaultListView"
    }
  },
  "facets" : {
    "asg.tagPathsHierarchy_ss" : {
      "label" : "Tags",
      "type" : "TREE",
      "values" : [ {
        "count" : 5,
        "value" : "BusinessApplication",
        "label" : "BusinessApplication",
        "checked" : false,
        "childValues" : [ {
          "count" : 5,
          "value" : "BusinessApplication/HDFS_BA",
          "label" : "HDFS_BA",
          "checked" : false,
          "childValues" : [ ]
        } ]
      }, {
        "count" : 5,
        "value" : "SC1_folderwith1subfolder",
        "label" : "SC1_folderwith1subfolder",
        "checked" : true,
        "childValues" : [ ]
      }, {
        "count" : 5,
        "value" : "Technology",
        "label" : "Technology",
        "checked" : false,
        "childValues" : [ {
          "count" : 5,
          "value" : "Technology/Big Data",
          "label" : "Big Data",
          "checked" : false,
          "childValues" : [ {
            "count" : 5,
            "value" : "Technology/Big Data/Hadoop Files",
            "label" : "Hadoop Files",
            "checked" : false,
            "childValues" : [ ]
          } ]
        } ]
      }, {
        "count" : 4,
        "value" : "createDir",
        "label" : "createDir",
        "checked" : false,
        "childValues" : [ ]
      } ]
    },
    "type_s" : {
      "label" : "Metadata Type",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 3,
        "value" : "Directory",
        "checked" : false
      }, {
        "count" : 2,
        "value" : "File",
        "checked" : false
      } ]
    },
    "asg.parentTypeNamePathHierarchy_ss" : {
      "label" : "Hierarchy",
      "type" : "TREE",
      "values" : [ {
        "count" : 7,
        "value" : "Cluster/Cluster Demo",
        "label" : "Cluster Demo [Cluster]",
        "checked" : false,
        "childValues" : [ {
          "count" : 6,
          "value" : "Cluster/Cluster Demo/Service/HDFS",
          "label" : "HDFS [Service]",
          "checked" : false,
          "childValues" : [ {
            "count" : 5,
            "value" : "Cluster/Cluster Demo/Service/HDFS/Directory/ROOT",
            "label" : "ROOT [Directory]",
            "checked" : true,
            "childValues" : [ {
              "count" : 4,
              "value" : "Cluster/Cluster Demo/Service/HDFS/Directory/ROOT/Directory/idaautomation",
              "label" : "idaautomation [Directory]",
              "checked" : true,
              "childValues" : [ {
                "count" : 3,
                "value" : "Cluster/Cluster Demo/Service/HDFS/Directory/ROOT/Directory/idaautomation/Directory/sub1",
                "label" : "sub1 [Directory]",
                "checked" : true,
                "childValues" : [ ]
              } ]
            } ]
          } ]
        } ]
      } ]
    },
    "asg_modifiedat_dt" : {
      "label" : "Last Modified",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 0,
        "value" : "(*,NOW-1YEAR]",
        "label" : "1 year and older",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-1YEAR,NOW-6MONTH]",
        "label" : "6 months up to 1 year",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-6MONTH,NOW-1MONTH]",
        "label" : "1 month up to 6 months",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-1MONTH,NOW-7DAY]",
        "label" : "1 week up to 1 month",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-7DAY,NOW-1DAY]",
        "label" : "1 day up to 1 week",
        "checked" : false
      }, {
        "count" : 5,
        "value" : "(NOW-1DAY,NOW]",
        "label" : "past 24 hours",
        "checked" : false
      } ]
    },
    "asg_modifiedby_s" : {
      "label" : "Modified by",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 5,
        "value" : "Service",
        "checked" : false
      } ]
    }
  }
}