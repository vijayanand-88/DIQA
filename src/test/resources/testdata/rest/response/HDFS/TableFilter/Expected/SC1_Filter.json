{
  "SC1": {
    "Directory": {
      "name": [
        "idaautomation",
        "sub1",
        "ROOT"
      ],
      "files": {
        "name": [
          "testdata.csv",
          "testdataRenamed.csv"
        ]
      }
    }
  },
  "SC2": {
    "Run1": {
      "Directory": {
        "name": [
          "idaautomation",
          "upload1",
          "ROOT"
        ],
        "files": {
          "name": [
            "uploadFile1.csv"
          ]
        }
      }
    },
    "Run2": {
      "Directory": {
        "name": [
          "idaautomation",
          "upload1",
          "upload2",
          "ROOT"
        ],
        "files": {
          "name": [
            "uploadFile1.csv",
            "uploadFile2.csv"
          ]
        }
      }
    }
  },
  "SC3": {
    "Run1": {
      "Directory": {
        "name": [
          "ROOT",
          "sub1",
          "sub7",
          "sub6",
          "asdsadfffffffffffffffffffffffffsaaaaaaaaaaaaaaaaaaaacdefghifklmnopqrstuvxyzqwertyuioasdfghjklzxcvbnmokmijnuhygvtfcrdxeszqawsxedcrfvtgbyhnujio",
          "idaautomation",
          "sub3",
          "sub2",
          "sub5",
          "sub4",
          "sub9",
          "sub8",
          "sub11",
          "sub12",
          "sub10"
        ],
        "files": {
          "name": [
            "userdata2.parquet",
            "uploadFile5.csv",
            "hdfs (1).zip",
            "userdata1.parquet",
            "insert_cycling1",
            "insert_cycling.txt",
            "uploadFile1.csv",
            "uploadFile4.csv",
            "Test+Results+-+Scenario__SC#33_2_Verify_Tag_is_set_for_the_column_when_namePattern_and_dataPattern_minimumRatio_matches_with_the_column_name_value_ratio_in_Oracle_table_ (4).html",
            "uploadFile3.csv",
            "insert_cycling_cassandra.txt",
            "dummy.pdf",
            "CALC-1985.doc"
          ]
        }
      }
    },
    "Run2": {
      "Directory": {
        "name": [
          "ROOT",
          "sub1",
          "sub7",
          "sub6",
          "asdsadfffffffffffffffffffffffffsaaaaaaaaaaaaaaaaaaaacdefghifklmnopqrstuvxyzqwertyuioasdfghjklzxcvbnmokmijnuhygvtfcrdxeszqawsxedcrfvtgbyhnujio",
          "idaautomation",
          "sub3",
          "sub2",
          "sub5",
          "sub4",
          "sub9",
          "sub8",
          "sub11",
          "sub10"
        ],
        "files": {
          "name": [
            "userdata2.parquet",
            "uploadFile5.csv",
            "hdfs (1).zip",
            "userdata1.parquet",
            "insert_cycling1",
            "insert_cycling.txt",
            "uploadFile1.csv",
            "uploadFile4.csv",
            "Test+Results+-+Scenario__SC#33_2_Verify_Tag_is_set_for_the_column_when_namePattern_and_dataPattern_minimumRatio_matches_with_the_column_name_value_ratio_in_Oracle_table_ (4).html",
            "uploadFile3.csv",
            "insert_cycling_cassandra.txt",
            "dummy.pdf",
            "CALC-1985.doc"
          ]
        }
      }
    }
  },
  "SC4": {
    "Run1": {
      "Directory": {
        "name": [
          "ROOT",
          "sub11",
          "sub10",
          "idaautomation",
          "asdsadfffffffffffffffffffffffffsaaaaaaaaaaaaaaaaaaaacdefghifklmnopqrstuvxyzqwertyuioasdfghjklzxcvbnmokmijnuhygvtfcrdxeszqawsxedcrfvtgbyhnujio",
          "sub1"
        ],
        "files": {
          "name": [
            "uploadFile1.csv",
            "uploadFile5.csv",
            "insert_cycling.txt",
            "insert_cycling1",
            "uploadFile4.csv"
          ]
        }
      }
    },
    "Run2": {
      "Directory": {
        "name": [
          "ROOT",
          "MaxHits",
          "Sub1"
        ]
      }
    }
  },
  "SC5": {
    "Directory": {
      "name": [
        "ROOT",
        "sub1",
        "sub7",
        "sub6",
        "asdsadfffffffffffffffffffffffffsaaaaaaaaaaaaaaaaaaaacdefghifklmnopqrstuvxyzqwertyuioasdfghjklzxcvbnmokmijnuhygvtfcrdxeszqawsxedcrfvtgbyhnujio",
        "idaautomation",
        "sub3",
        "sub2",
        "sub5",
        "sub4",
        "sub9",
        "sub8",
        "sub11",
        "sub10"
      ],
      "files": {
        "name": [
          "userdata2.parquet",
          "hdfs (1).zip",
          "userdata1.parquet",
          "insert_cycling1",
          "insert_cycling.txt",
          "Test+Results+-+Scenario__SC#33_2_Verify_Tag_is_set_for_the_column_when_namePattern_and_dataPattern_minimumRatio_matches_with_the_column_name_value_ratio_in_Oracle_table_ (4).html",
          "insert_cycling_cassandra.txt",
          "dummy.pdf",
          "CALC-1985.doc"
        ]
      },
      "Cluster": {
        "name": [
          "Sandbox"
        ]
      }
    }
  },
  "SC6": {
    "Directory": {
      "name": [
        "ROOT",
        "idaautomation",
        "unique",
        "f2_s1_s1",
        "f1_s1",
        "f2_s1",
        "F1",
        "f2",
        "f2",
        "f2_s1"
      ],
      "files": {
        "name": [
          "Test+Results+-+Scenario__SC#33_2_Verify_Tag_is_set_for_the_column_when_namePattern_and_dataPattern_minimumRatio_matches_with_the_column_name_value_ratio_in_Oracle_table_ (4).html",
          "uploadFile3.csv",
          "uploadFile1.csv",
          "uploadFile3.csv"
        ]
      }
    }
  },
  "SC6_1": {
    "Directory": {
      "name": [
        "ROOT",
        "Sub4",
        "Sub3",
        "Sub2",
        "Sub1",
        "MaxHits"
      ],
      "files": {
        "name": [
          "uploadFile2.csv"
        ]
      }
    }
  },
  "SC7": {
    "Directory": {
      "name": [
        "ROOT",
        "HDFSMonitor_Test",
        "NewFolder"
      ],
      "files": {
        "name": [
        ]
      }
    }
  },
  "SC8": {
    "Directory": {
      "name": [
        "ROOT",
        "HDFSMonitor_Test",
        "NewFolder",
        "Sub-Dir",
        "NewFolder1"
      ],
      "files": {
        "name": [
        ]
      }
    }
  },
  "SC9": {
    "Directory": {
      "name": [
        "ROOT",
        "HDFSMonitor_Test",
        "NewFolder",
        "Sub-Dir",
        "NewFolder1"
      ],
      "files": {
        "name": [
          "userdata1.parquet",
          "hdfs (1).zip",
          "uploadFile1.csv",
          "dummy.pdf",
          "CALC-1985.doc",
          "insert_cycling_cassandra.txt"
        ]
      }
    }
  },
  "SC10": {
    "Directory": {
      "name": [
        "ROOT",
        "HDFSMonitor_Test",
        "NewFolder",
        "Sub-Dir",
        "NewFolder1"
      ],
      "files": {
        "name": [
          "userdata1.parquet",
          "hdfs (1).zip",
          "uploadFile1.csv",
          "dummy.pdf",
          "CALC-1985.doc",
          "insert_cycling_cassandra.txt"
        ]
      }
    }
  }
}

